KP5-Topic2/KP1-Topic2/src/test/java/com/aguirre/KP1_Topic2/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package com.aguirre.KP1_Topic2;
```

Comment:
Why did you added this test class? You are not really testing the App class or am I wrong?